File: Ad-Hoc/1329/1329.c
#include <stdio.h>

int main(){
    int n, cara = 0, coroa = 0, x;
    scanf("%d", &n);
    while (n != 0){
        for (int i = 0; i < n; i++){
            scanf("%d", &x);
            if (x == 0){
                cara++;
            }
            else{
                coroa++;
            }
        }
        printf("Mary won %d times and John won %d times\n", cara, coroa);
        cara = 0;
        coroa = 0;
        scanf("%d", &n);
    }
    return 0;
}

File: Ad-Hoc/1329/1329.py
n = int(input())
cara = 0
coroa = 0
while n!=0:
    x = [int(i) for i in input().split()]
    cara = x.count(0)
    coroa = x.count(1)
    print(f"Mary won {cara} times and John won {coroa} times")
    cara = 0
    coroa = 0
    n = int(input())